lib/interpreters/compute_several_solutions.rb

```diff
@@ -214,7 +214,7 @@ def self.find_best_heuristic(service_vrp)
             }.compact
             next if mission_ids.empty?
 
-            Models::Route.new(vehicle: vrp.vehicles.find{ |v| v[:id] = route[:vehicle_id] }, mission_ids: mission_ids)
+            Models::Route.new(vehicle: vrp.vehicles.find{ |v| v[:id] == route[:vehicle_id] }, mission_ids: mission_ids)
```

Comment:
:face_with_head_bandage:  this is the second time I introduced this exact same error and it ended up on beta :(